src/app.ts

```diff
@@ -51,4 +55,32 @@ const listener = (): void => {
   )
 }
 
+const server = https.createServer(
+  {
+    SNICallback: (domain, cb) => {
+      // using whoami will return root because app is run with sudo
+      const adminUser = execSync('logname')
+        .toString()
+        .trim()
+      // escape characters required or readFileSync will not find file
+      const secureContext = tls.createSecureContext({
+        /* eslint-disable */
+        key: fs.readFileSync(
+          `/home/${adminUser}/\.acme\.sh/*\.${domain}/*\.${domain}\.key`
```

Comment:
instead of `/home/${adminUser}`, could you do `~/` instead? Because for `root` user, the home path is `/root`